library/dokku_ports.py

```diff
@@ -57,13 +57,25 @@
 
 def dokku_proxy_port_mappings(data):
     mappings = []
-    command = "dokku --quiet proxy:report {0}".format(data["app"])
+
+    if use_legacy_command():
+        command = "dokku --quiet proxy:report {0}".format(data["app"])
+    else:
+        command = "dokku --quiet ports:report {0} --ports-map".format(data["app"])
+
     output, error = subprocess_check_output(command)
     if error is None:
-        for line in output:
-            match = re.match("Proxy port map:(?P<mapping>.+)", line.strip())
-            if match:
-                mappings = match.group("mapping").strip().split(" ")
+        if use_legacy_command():
+            for line in output:
+                match = re.match("Proxy port map:(?P<mapping>.+)", line.strip())
+                if match:
+                    mappings = match.group("mapping").strip().split(" ")
```

Comment:
this is not your bug but it seems to me, `mappings` here is never set if no match is found.
could you fix this if you don't mind?

Reply:
You know what, `mappings = []` actually already was initialized, I just didn't see it.

https://github.com/dokku/ansible-dokku/pull/166/files#diff-771a074fd2105a1b2651c2dec4070b48246b5ec753f5824031bf3015d07a7f3cR59